go.sum

```diff
@@ -209,6 +209,10 @@ github.com/containers/conmon v2.0.2+incompatible h1:h2HCdd/EBpwFn7RT82Y2GyXnVUHW
 github.com/containers/conmon v2.0.2+incompatible/go.mod h1:hgwZ2mtuDrppv78a/cOBNiCm6O0UMWGx1mu7P00nu5I=
 github.com/containers/conmon v2.0.3+incompatible h1:jxH9Nm1owQ8spsmISHJGiUChqTKrw9BUuEhdR8196bE=
 github.com/containers/conmon v2.0.3+incompatible/go.mod h1:hgwZ2mtuDrppv78a/cOBNiCm6O0UMWGx1mu7P00nu5I=
+github.com/containers/conmon v2.0.6+incompatible h1:FCnBv2AnxClfvpRlg5moKYPi2pBFVLRUfsoaql2z59s=
```

Comment:
why are these piling up? :/

Reply:
https://github.com/renovatebot/renovate/issues/2594#issuecomment-426507432
kind of gives an answer